### PR TITLE
live radio int tests tidy up

### PR DESCRIPTION
**Pull request by jroebu14**
**Overall change:**
Tidy up of int tests for live radio page. I wanted to carry over some of the tidy ups I made in this [draft PR](https://github.com/bbc/simorgh/pull/6626/files). I'd like to do this page by page. Here are the changes for the live radio page type

- Headline is now a reusable set of tests
- Snapshots make more sense when viewing e.g. `AMP Amharic Live Radio Page Main Heading should match text  "ያድምጡ"`
- separate it blocks for assertions make tests clearer e.g.
```js
    it('should be in the document', () => {
      expect(summaryEl).toBeInTheDocument();
    });

    it('should contain text', () => {
      expect(summaryText).toBeTruthy();
    });

    it('should match text', () => {
      expect(summaryText).toMatchSnapshot();
    });
```
- Simpler test file setup in, for example, `src/integration/pages/liveRadio/korean/canonical.test.js`

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/integration/common/headline.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default () => {
+  describe('Headline', () => {
```
Comment (greenc05) [inline src/integration/common/headline.js]: What is headlline? The h1?

Comment (jroebu14) [reply src/integration/common/headline.js]: @greenc05 yep

Comment (jroebu14) [reply src/integration/common/headline.js]: is `Main Heading` better?

Comment (greenc05) [reply src/integration/common/headline.js]: And the reason we call this headline rather than h1 or headingLevel1 etc?

Comment (greenc05) [reply src/integration/common/headline.js]: Sorry, commenting at the same time. It's the h1 ya? So IMO I would call it what it is.

Comment (jroebu14) [reply src/integration/common/headline.js]: I try to think of naming components from a user perspective and not so low-level. A user and possibly a test engineer would call it a headline, title, or main heading. I'd rather go with that

Comment (greenc05) [reply src/integration/common/headline.js]: Headline could be any heading. Title could be the page title. Main heading is the best option out of those IMO.

Comment (jroebu14) [reply src/integration/common/headline.js]: I agree. Thanks!

Comment (jroebu14) [reply src/integration/common/headline.js]: done!

**Review by karinathomasbbc — APPROVED** [review]
Really nice, thanks for this :shipit:

**Review by rebeccamcginn — APPROVED** [review]
Still my favourite: https://giphy.com/gifs/1rNzWL6GFvGR9epyij

His eyes as he realises...

**Pull request merged**